### ISSUE TITLE
homepage - totals box color

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#63]
https://omeka.religiousecologies.org/s/census-1926/page/home

I think the totals data box here should have a black box border instead of the blue one as it visually looks like an image link next to the others, so it may confuse the user. 

<img width="649" alt="Screen Shot 2020-12-07 at 9 26 57 AM" src="https://user-images.githubusercontent.com/618307/101362903-ba6a0380-386e-11eb-8d50-9d111ca5ad3d.png">

**Comment by kimisgold** [comment]
This is live now. Let me know if this looks better.

**Comment by IdaAkiwumi** [comment]
Much better! Thank you!
